config.js

```diff
@@ -33,74 +33,91 @@ displaySystem.config({
             ],
         },
         'clock': {
-            visible: true,
+            visible: false,
             // countdown: 20,
         },
         'time': {
-            visible: true,
+            visible: false,
             // format: "MM:ss",
         },
         'twitter': {
             // visible: true,
             // speed: 500,
         },
         'lowThird': {
-            visible: true,
+            visible: false,
             line1: '❤️ press \'C\'',
             line2: 'to show the control window',
         },
         'list': {
             // visible: true,
             header: 'results round 1',
             data: [
-                ['🇳🇱 just try',1],
-                ['🇳🇱 to copy',2],
-                ['🇪🇬 and paste',3],
-                ['🇪🇬 some cells',4],
-                ['🇪🇬 from excel',5],
-                ['🇺🇸 into the control',6],
-                ['🇺🇸 window! You\'ll be',7],
-                ['🇺🇸 amazed!',8],
-                ['🇳🇱 It even has',8],
-                ['🇳🇱 automatic',8],
-                ['🇳🇱 paging',8],
+                ['🇳🇱 just try', 1],
+                ['🇳🇱 to copy', 2],
+                ['🇪🇬 and paste', 3],
+                ['🇪🇬 some cells', 4],
+                ['🇪🇬 from excel', 5],
+                ['🇺🇸 into the control', 6],
+                ['🇺🇸 window! You\'ll be', 7],
+                ['🇺🇸 amazed!', 8],
+                ['🇳🇱 It even has', 8],
+                ['🇳🇱 automatic', 8],
+                ['🇳🇱 paging', 8],
             ],
             timer: 5000,
-            lines: 8
+            lines: 12
         },
         'sprite': {
-            // visible: true,
+            visible: true,
+            texts: [{
+                data:'2017 Regional, Modiin'
+            }
+            ],
             sprites: [{
-                width: '100%',
-                top: '3.5vh',
-                fontSize: '8vh',
-                textAlign: 'center',
-                color: 'rgba(255,255,255,0.5)',
-                html: 'FIRST LEGO League'
+                side: 'top',

```

Comment:
These are too specific for one tournament, I'd like to keep the default configuration lightweight and easily adjustable